Workbooks/CosmosDb/Overview/Overview.workbook

```diff
@@ -436,11 +464,402 @@
             "itemKey": "$gen_heatmap_microsoft.documentdb/databaseaccounts-Requests-TotalRequestUnits_5",
             "sortOrder": 2
           }
+        ]
+      },
+      "conditionalVisibility": {
+        "parameterName": "selectedTab",
+        "comparison": "isEqualTo",
+        "value": "Overview"
+      },
+      "showPin": true,
+      "name": "cosmosDB account metrics"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "# Overview section ends here\r\n <br />  <br />"
+      },
+      "conditionalVisibility": {
+        "parameterName": "dummy",
+        "comparison": "isEqualTo",
+        "value": "blah"
+      },
+      "name": "text - 4 - Copy"
```

Comment:
steps should ideally have useful names instead of copy of copy of copy

Reply:
fixed